homeassistant/components/kitchen_sink/backup.py

```diff
@@ -23,6 +25,8 @@ async def async_get_backup_agents(
 class KitchenSinkBackupAgent(BackupAgent):
     """Kitchen sink backup agent."""
 
+    domain = DOMAIN
```

Comment:
I think we should update the kitchen sink platform with the new protocol function and write tests for it.